packages/slice-machine/src/pages/changes.tsx

```diff
@@ -86,7 +101,10 @@ const Changes: React.FunctionComponent = () => {
         pushChangesSuccess();
 
         setIsPushed(true);
-        toast.success("All slices and types have been pushed");
+
+        isPromptToCreateContentExperimentEligible
+          ? setIsToastOpen(true)
+          : toast.success("All slices and types have been pushed");
```

Comment:
💡 Ternaries are not very readable to perform actions, only for assignments IMO. A regular if statement would look better here. WDYT?

```suggestion
        if (isPromptToCreateContentExperimentEligible) {
          setIsToastOpen(true);
        } else {
          toast.success("All slices and types have been pushed");
        }
```

Reply:
Don't have strong opinions here, will change it to use if statement